README.md

```diff
@@ -20,15 +20,16 @@ $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Minifor
 # For MacOS:
 # $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh
 
-$ bash ./Miniforge3-Linux-x86_64.sh
+$ sh ./Miniforge3-Linux-x86_64.sh
 # make sure the installer runs 'conda init' at the end of installation, or run it manually.
 
 # Restart shell to load conda integration
-$ exec bash
+$ exec $SHELL
 
 # Optional: create conda environment
-$ conda create -n dgfem
-$ conda activate dgfem
+$ export MY_CONDA=/path/to/installed/conda # Default installation path: $HOME/miniforge3
+$ $MY_CONDA/bin/conda create -n dgfem
+$ $MY_CONDA/bin/conda activate dgfem
```

Comment:
There should be a `source` here. Could you please make sure to propose tested changes?

Reply:
Well, the instructions as shown in this patch worked for me. Changing to `.`, as `source` is bash-only.

Reply:
> Well, the instructions as shown in this patch worked for me.

It may not have failed, but unless you had the `dgfem` environment activated already, I doubt that it was activated after this command.